include/deal.II/matrix_free/evaluation_kernels_face.h

```diff
@@ -1447,116 +1447,110 @@ namespace internal
   template <int dim, typename Number>
   struct FEFaceEvaluationImplEvaluateSelector
   {
-    template <int fe_degree, int n_q_points_1d>
     static bool
-    run(const unsigned int                     n_components,
-        const EvaluationFlags::EvaluationFlags evaluation_flag,
-        const Number                          *values_dofs,
-        FEEvaluationData<dim, Number, true>   &fe_eval)
+    evaluate_tensor_none(const unsigned int                     n_components,
```

Comment:
private?

Reply:
We could make this private but this is internal anyway. Would you be tempted to use it in one of your codes otherwise? :wink: